keptn-cert-manager/pkg/certificates/watcher.go

```diff
@@ -135,3 +135,14 @@ func (watcher *CertificateWatcher) ValidateCertificateExpiration(certData []byte
 	}
 	return true, nil
 }
+
```

Comment:
looks good! Can we please move the changes in the `keptn-cert-manager` insto a new separate PR? This way we can point the `keptn-cert-manager` dependencies to the keptn repository, as now it points to your fork (this way we cannot merge that). These changes  (in `keptn-cert-manager`) can be pushed directly to `main` branch instead of `epic/cert-manager.io` branch

Reply:
Alright let me do that now